src/input-nested-select/input-nested-select.tsx

```diff
@@ -45,7 +41,10 @@ export const InputNestedSelect = <V1, V2, V3>({
     // =============================================================================
     // CONST, STATE
     // =============================================================================
-    const [selected, setSelected] = useState<V1 | V2 | V3>(selectedOption);
+    const [selectedKey, setSelectedKey] = useState<string[]>(selectedKeyPath);
```

Comment:
is it without 's'? eg: `selectedKeyPath`

once multiple coming in it would be `selectedKeyPaths: string[][]`

Reply:
is it without `s`? eg: `selectedKeyPath`

we have `selectedKeyPath` and `selectedKeyPaths` once multi selection was been updated:

```
interface DrodpownInterface {
   ...,
  selectedKeyPath: string[],
  selectedKeyPaths: string[][],
 ....,
}
```

Reply:
I was thinking it could be a single prop, so the local state in nested select would be `string[][]` too. but ok keep as `selectedKeyPath` without the s for now

```
interface DropdownInterface {
  selectedKeyPaths: string[][],
  ...
}
```